corehq/form_processor/migrations/0060_convert_case_ids_to_foreign_keys.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+from corehq.sql_db.operations import HqRunSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('form_processor', '0059_remove_ledgervalue_location_id'),
+    ]
+
+    operations = [
+        # The other way to convert a field to a foreign key involves creating a new column
+        # doing a data migration and then dropping the old column which seemed ridiculous to me
+        HqRunSQL(
+            'ALTER TABLE "form_processor_ledgervalue" '
+            'ADD CONSTRAINT "cd40c15ceaad5d793e09d0b69eb4ed88" FOREIGN KEY ("case_id") '
+            'REFERENCES "form_processor_commcarecasesql" ("case_id") DEFERRABLE INITIALLY DEFERRED',
```

Comment:
I got this SQL by doing a running `sqlmigrate` command with the state operations below. The name of the constraints are consistent across environments.